android/app/src/main/res/drawable/ic_bootsplash.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="100dp"
+    android:height="100dp"
+    android:viewportWidth="100"
+    android:viewportHeight="100">
+  <path
+      android:pathData="M74,15.63a42.03,42.03 0,0 0,-24 -7.5A41.77,41.77 0,0 0,8.12 49.87c0,9 2.76,17.26 7.5,24a4.35,4.35 0,0 1,-0.37 5.63c-1.75,1.63 -4.5,1.5 -5.88,-0.38A49.99,49.99 0,0 1,50 0c10.88,0.01 20.88,3.51 29.25,9.38 2,1.5 2,4.26 0.38,5.88a4.35,4.35 0,0 1,-5.63 0.38ZM67.62,24.87L67.62,35.5L45.25,35.5v9.13h18v10.62h-18v9.63L67.5,64.88v10.37L32.25,75.25L32.25,24.87h35.38ZM91.12,21.75a3.78,3.78 0,0 0,-5.87 -0.38,4.35 4.35,0 0,0 -0.5,5.5c4.38,6.63 7,14.63 7,23.13 0,8.5 -2.63,16.75 -7.25,23.5 -1.13,1.75 -1.13,4.13 0.38,5.63 1.75,1.75 4.5,1.62 5.87,-0.38A49.5,49.5 0,0 0,99.88 50a47.66,47.66 0,0 0,-8.75 -28.25ZM26.88,84.88c6.63,4.37 14.63,7 23.13,7 8.63,0 16.75,-2.75 23.5,-7.25a4.35,4.35 0,0 1,5.63 0.37c1.75,1.75 1.62,4.5 -0.38,5.88A50.37,50.37 0,0 1,50 100a49.75,49.75 0,0 1,-28.25 -8.75c-2,-1.38 -2.13,-4.13 -0.38,-5.88 1.38,-1.5 3.75,-1.62 5.5,-0.5Z"
+      android:fillColor="#002E22"
+      android:fillType="evenOdd"/>
+</vector>
```

Comment:
Is this file still necessary? I think it might not be used since we reverted the splash changes.

Reply:
Nope, missed that file, I'm going through all files and checking if I didn't miss anything else.